collect_app/src/main/java/org/odk/collect/android/widgets/AudioController.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (C) 2017 Shobhit
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.odk.collect.android.widgets;
+
+import android.content.Context;
+import android.media.AudioManager;
+import android.media.MediaPlayer;
+import android.net.Uri;
+import android.os.Handler;
+import android.view.View;
+import android.widget.ImageButton;
+import android.widget.SeekBar;
+import android.widget.TextView;
+
+import org.odk.collect.android.R;
+import org.odk.collect.android.activities.FormEntryActivity;
+import org.odk.collect.android.utilities.MediaPlayerUtilities;
+
+import java.io.File;
+import java.io.IOException;
+
+import timber.log.Timber;
+
+class AudioController implements View.OnClickListener, SeekBar.OnSeekBarChangeListener {
+    private final Context context;
+    private final MediaPlayer mediaPlayer;
+
+    private ImageButton playButton;
+
+    private SeekBar seekBar;
+
+    private TextView totalDurationLabel;
+    private TextView currentDurationLabel;
+
+    private int seekForwardTime = 5000; // 5000 milliseconds
```

Comment:
What do you think about calling these two variables `seekForwardMillis` and `seekBackwardMillis` ? I think their intentions might be clearer and they be self explanatory so the comments wouldn't be necessary. Just a thought.